shell/utility.go

```diff
@@ -55,13 +69,16 @@ func parseJSON(b []byte) (map[string]string, error) {
 	return output, err
 }
 
-func runCommand(command string, state *State, environment []string, workingDirectory string) (*State, error) {
+func runCommand(command string, state *State, environment []string, workingDirectory string, interpreter [2]string) (*State, error) {
 	const maxBufSize = 8 * 1024
 	// Execute the command using a shell
 	var shell, flag string
 	if runtime.GOOS == "windows" {
 		shell = "cmd"
 		flag = "/C"
+	} else if interpreter[0] != "" {
```

Comment:
Wrong logic: the custom interpreter won't be used on windows. It needs to have top priority!

Reply:
Interpreter priority changed